src/App.scss

```diff
@@ -0,0 +1,13 @@
+@use './styles/setup.scss';
+@use './styles/global.scss';
+@use './styles/visuals.scss';
+@use './styles/variables.scss';
+@use './styles/questionBox.scss';
+@use './styles/ship.scss';
+@use './styles/bodyDiv.scss';
+@use './styles/media.scss';
+
+.App {
+  text-align: center;
+  height: 100vh;
+}
```

Comment:
great use of partials to organize the styling and keep everything from getting unwieldy/overwhelming for your styles!